### ISSUE TITLE
[QUESTION] What does "User %s is attempting recursion" mean?

### ISSUE THREAD

**Issue by msr8** [trambelus/UserSim#12]
So I was going through the source code, and saw [these](https://github.com/trambelus/UserSim/blob/4c408f0eee4321acc52209a0916131d3237cd32b/usim.py#L266-L267) lines, but I dont get what they do or what the "recursion" feature is

**Comment by REK166** [comment]
It's supposed to stop the bot from answering to itself,
because people might try to get the bot to say something that will be registered as A prompt to the bot, creating an infinite loop and DOSing the bot.

**Comment by trambelus** [comment]
True, and to add to that, the bot was already designed in a way that
wouldn't allow it to respond to itself and create loops like that. I just
noticed certain people trying it, so I wrote in that response as a sort of
easter egg. :)
